1-exercises/B-highlight-words/script.js

```diff
@@ -1,10 +1,54 @@
 function highlightWords(paragraph, colours) {
-  // Write your code here...
+const div = document.querySelector('#content');
+
+//creating a 'p' tag for paragraf 
+const p = document.createElement('p');
+//creating button select
+const select = document.createElement("select");
+
+//displaying "p" in 'div'
+div.appendChild(p);
+
+//turning the paragraph into an array
+  paragraph.split(" ").forEach((word) => {
+    //creating "span" (marker)
+    const span = document.createElement("span");
+    //when cursor is click to the word background-color is changing to selected value
+    span.addEventListener("click", (event) => {
+      //
+      const selectedColour = document.querySelector("select").value;
+      event.target.style.backgroundColor = selectedColour;
+    });
+    span.innerText = word + " ";
+    p.appendChild(span);
+  });
+
+
+//displaying"select" in 'div'
+div.appendChild(select);
+//using spread operator and foEach method to set color options
+[...colours].forEach((colour) => {
```

Comment:
Spread operator is not necessary, instead: `colours.forEach((color) => {` would be enough